### PR TITLE
test: remove x-ray from example tests and add LAMBDA_TASK_ROOT

### PR DESCRIPTION
**Pull request by AlexeySoshin**
Add a segment before the test starts and end it once it finishes, same as with Maven examples

**Issue #, if available:**

## Description of changes:

Other examples have setup/teardown functions
https://github.com/aws-powertools/powertools-lambda-java/blob/ef11a64df17ee1cbf736aacbd308c9f191d21a8c/examples/powertools-examples-core/sam/src/test/java/helloworld/AppTest.java#L30

**Checklist**
* [ ] [Meet tenets criteria](https://docs.powertools.aws.dev/lambda-java/#tenets)
* [ ] Update tests
* [ ] Update docs
* [ ] PR title follows [conventional commit semantics](https://www.conventionalcommits.org/en/v1.0.0/)

## Breaking change checklist

**RFC issue #**:

* [ ] Migration process documented
* [ ] Implement warnings (if it can live side by side)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by jeromevdl** [comment]
To be frank, I wanted to remove tests in samples, but maybe we should have better tests instead.

The tracing stuff is typically the kind of boring things. Idempotency with dynamodb also (we had a question about that), ...

And it's funny because we have samples containing code that we don't recommend: https://docs.powertools.aws.dev/lambda/java/core/tracing/#configure-test-cases-not-recommended

**Comment by scottgerring** [comment]
I thought we should just merge this for consistency, but now I am less sure 🤣 I think it is 1/ nice that we have a test that exercises the handler but 2/ think the x-ray stuff is a bit overwrought. A third option would be keeping the tests and removing the x-ray tracing from all of them, which lines us up with our documented best practices.

@jeromevdl ?

**Comment by jeromevdl** [comment]
then we must add the property in the pom.xml: https://docs.powertools.aws.dev/lambda/java/core/tracing/#configure-environment-variable-on-project-level-recommended

**Comment by AlexeySoshin** [comment]
@jeromevdl I have added this to the Gradle example for now:
https://github.com/aws-powertools/powertools-lambda-java/pull/1451/files#diff-80b08d79d09d03fb438f451205323a1e77e99ba8ffe4842545d235b5b2790989R37

**Comment by scottgerring** [comment]
@AlexeySoshin I think you should remove x-ray from all the example tests (right @jeromevdl ? )

File: examples/powertools-examples-core/gradle/build.gradle [hunk 1]
```diff
@@ -11,10 +11,6 @@ wrapper {
 compileJava {
     sourceCompatibility = "1.8"
     targetCompatibility = "1.8"
-
-    ajc {
```
Comment (AlexeySoshin) [inline examples/powertools-examples-core/gradle/build.gradle:15]: @scottgerring FYI, I think this is not needed as you already use `aspect`

Comment (scottgerring) [reply examples/powertools-examples-core/gradle/build.gradle:15]: If it works without it let's get rid of it!

**Comment by jeromevdl** [comment]
we removed all the tests in the example, I think we can close it.

**Pull request closed without merging**